sndtimer/sndtimer.js

```diff
@@ -52,7 +53,7 @@ handle_file_change = (event) => {
             player_forward.classList.add("btn-secondary");
         }
 
-        player_timestamp.innerHTML = ticks_to_timestring(offset);
+        player_timestamp.innerHTML = ticks_to_timestring(offset, false);
```

Comment:
It looks like this is a remnant of the test idea you removed in ee90369a17ce519ef515111c6328602ba1845331 - is it safe to remove the false parameter from here?